### DISCUSSION TITLE
Tailwind merge or loading the tailwind classes in module federation using vite and react

### DISCUSSION THREAD

**Discussion by lauraDiazHahnSoftware** [tailwindlabs/tailwindcss#15316] [Help]
I'm having an issue with tailwind and module federation that stops me to proper configure tailwind in the shell or host app. What is happening is that I have my shell app with tailwind theme and styles and when I register and load my remotes using module federetaion runtime for dynamic loading I don't get full styles because the shell don't have compiled all the classes from tailwind and didn't compile this new added so style are partially broken, a side solution maybe to add to the file that contains the remote component I'm sharing the tailwindcss import, this fix the styles for the remote in the shell but it's messing up the shell app styles. 
Does someone have a solution for this?

**Comment by lucasbrittos** [comment]
I've been facing the same problem here. If you get any solution, please share it.

**Comment by GlazGo31** [comment]
@lauraDiazHahnSoftware Hi. Please share a solution to this problem if you have one.
